tests/h/services/list_groups_test.py

```diff
@@ -112,6 +112,24 @@ def test_sorts_groups_alphabetically(self, svc, user, mixed_groups):
 
         assert [group.pubid for group in results] == expected_sorted_pubids
 
+    def test_returns_associated_groups_if_user(self, svc, authority, user, scoped_open_groups, unscoped_open_groups, private_groups):
+        user.groups = private_groups
+        expected_ids = {group for group in (scoped_open_groups + unscoped_open_groups + private_groups)}
```

Comment:
It confused me here for a moment to have a variable named `expected_ids` which actually contains a set of `Group`s. Can you change this?